packages/cli/src/cmds/assert/strategy/httpRequestStrategy.ts

```diff
@@ -0,0 +1,19 @@
+import Strategy from './strategy';
+import { AppMapData } from '../../../appland/types';
+import Assertion from '../assertion';
+
+export default class HttpRequestStrategy implements Strategy {
+  check(appMap: AppMapData, assertion: Assertion): Boolean {
+    for (let e of appMap.events) {
+      if (e.hasOwnProperty('http_server_response') && !eval(assertion.assert)) {
```

Comment:
`Event` has a bunch of built-in functions that you can use, for example `e.httpServerRequest` rather than digging in with `hasOwnProperty`